README.md

```diff
@@ -107,19 +105,13 @@ curl \
   -H "X-Auth: $token" \
   -H "Content-Type: application/json" \
   -H "Accept: application/json" \
-  -X POST http://try-runtime-bot/api/queue \
+  -X POST http://command-bot/api/queue \
   -d '{
-    "execPath": "cargo",
-    "args": [
-      "run",
-      "--quiet",
-      "--features=try-runtime",
-      "try-runtime",
-      "--block-at=0x5d67782862757220cb25cf073585f6a75a9031f1da4115e5cba1721c2c6e249c",
-      "--url=ws://polkadot",
-      "on-runtime-upgrade",
-      "live"
-    ],
+    "job": {
+      "tags": ["linux-docker"],
+      "image": "paritytech/ci-linux:production"
```

Comment:
This seem insecure to be able to run any docker image like that from the command-bot directly. Also it will not be able to spawn docker containers from where it's running in Kubernetes pods. It's just not allowed there.

Reply:
> seems like it would be available by an unauthentified api cal

The API requires authentication as shown in the `X-Auth: $token` part of the example

> but overriding the docker image can be a way to introduce malicious code

Specifying the image can only be done by API calls which is authenticated by the token. Again it's a matter of if it's worthwhile to guard the bot against its own users. I don't think so.

And in case it's not clear, this is specifying the image for the CI runner's job, not the bot itself. It's currently possible and quite trivial to do this "malicious image injection" in any project we have by editing the `.gitlab-ci.yml` in a pull request (to be improved by https://gitlab.parity.io/parity/opstooling/gitlab-guard).